scripts/reindex_with_types.py

```
@@ -0,0 +1,278 @@
+        Path("/app/cache/.file_search_cache.json"),  # Docker path
+    ]
+
+    for cache_path in cache_paths:
+        if cache_path.exists():
+            if dry_run:
+                logger.info(f"[DRY RUN] Would delete cache: {cache_path}")
+            else:
+                try:
+                    cache_path.unlink()
+                    logger.info(f"Deleted cache file: {cache_path}")
+                except Exception as e:
```

Ruff (0.14.10)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)